modules/apps/accessibility/accessibility-menu-web/src/main/java/com/liferay/accessibility/menu/web/internal/display/context/AccessibilityMenuDisplayContext.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.accessibility.menu.web.internal.display.context;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import com.liferay.accessibility.menu.web.internal.util.AccessibilitySettingsUtil;
+
+import java.util.List;
+
+import javax.servlet.http.HttpServletRequest;
+
+/**
+ * @author Evan Thibodeau
+ */
+public class AccessibilityMenuDisplayContext {
+
+	public AccessibilityMenuDisplayContext(
+		HttpServletRequest httpServletRequest) {
+
+		_httpServletRequest = httpServletRequest;
+	}
+
+	public List<Object> getAccessibilitySettings() {
+		ObjectMapper objectMapper = new ObjectMapper();
+
```

Comment:
Hey @ethib137, please use the `JSONUtil.toJSONArray` instead of the jackson ObjectMapper. For your use case using the JSONUtil is enough. In the end, your code will look like this:
```
public JSONArray getAccessibilitySettingsJSONArray() {
      return JSONUtil.toJSONArray(
           AccessibilitySettingsUtil.getAccessibilitySettings(_httpServletRequest),
           accessibilitySetting -> {
               //blablabla
           }
      )
}
```

Here is an example https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/display/context/FDSViewsDisplayContext.java#L52-L66

Reply:
Sounds good.. thanks.